src/app/shared/detail/detail-container/detail-container.component.html

```diff
@@ -1 +1,10 @@
-<p>detail-container works!</p>
+<mu-header></mu-header>
+<article>
+  <section class="article-container">
+    <img class="article-profile-image" src="{{heroData.thumbnail.path}}.{{heroData.thumbnail.extension}}"
+         alt="profile_image">
+    <div *ngIf='heroData.description !=""; else no_desc' class = "hero-desc">{{heroData.description}}</div>
```

Comment:
since your template does not have any styles as such, you dont need an ngIf or a template..

there is another way to handle this in text interpolation. We can use conditions within {{}}